doc/chapters/integration-guidelines/10-traffic-shaping.rst

```diff
@@ -0,0 +1,72 @@
+Traffic Shaping
+===============
+
+Linux offers a very rich set of tools for managing and manipulating the transmission of packets. And
+since the host system is Linux based, those tools can be used to manage and shape traffics on Software
+Containers. Linux community is very familiar with the tools available under Linux for packet mangling and
+firewalling (netfilter, and before that, ipchains) as well as hundreds of network services which can
+run on the operating system.
+
+Linux community are aware of the tremendous power of the traffic control subsystem which has grown
+and matured. To glace those variaty of solutions the linux documentation project would be very good
```

Comment:
glace? variaty?

I'm not sure we need to mention that the Linux community is aware etc, but might be worth mentioning that it is very flexible. Something like:

"The Linux network routing, firewalling and traffic control subsystem is very powerful and flexible and have grown to be a very mature stack of tools." Not sure if thats perfect either though ;)